pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -4386,7 +4386,9 @@
         "4.5.0"
       ],
       "BaselineVersion": "4.4.0",
-      "InboxOn": {},
+      "InboxOn": {
+        "uap10.0.16299": "4.0.1.0"
```

Comment:
shouldn't this be vNext? so like uap10.0.16300?

Reply:
Yep you're absolutely right, I'll fix that.

Reply:
@joperezr I just noticed that System.ComponentModel.Composition doesn't have a uap10.0.16300 entry. I will have to add that in, right?

Reply:
mm in theory that should be ok since I believe we didn't change the assembly version, so the way this index works is that if you don't change an assembly version then you don't need an extra InboxOn clause.

Reply:
Ok, so at this point do these changes look good? Unfortunately we do expect the tests to not come back clean in this branch...